Source/Extensions/Blazorise.QRCode/QRCode.cs

```diff
@@ -133,6 +160,26 @@ private static byte[] GetColorBytes( string color )
         /// </summary>
         [Parameter] public bool DrawQuietZones { get; set; } = true;
 
+        /// <summary>
+        /// The icon that is places in the middle of the QRCode, in base64 format.
+        /// </summary>
+        [Parameter] public string Icon { get; set; }
+
+        /// <summary>
+        /// Defines how much space the icon will occupy within the QRCode.
+        /// </summary>
+        [Parameter] public int IconSizePercent { get; set; } = 15;
+
+        /// <summary>
+        /// Defines how large the borders will be for the icon that is placed within the QRCode.
+        /// </summary>
+        [Parameter] public int IconBorderWidth { get; set; } = 6;
+
+        /// <summary>
+        /// Defines the icon image file format.
+        /// </summary>
+        [Parameter] public IconImageType IconImageType { get; set; } = IconImageType.Png;
```

Comment:
Would we want this to be string? Wouldn't SVG, IMG, etc... also work? What if user wants to use an img we haven't added to the enum.
While at least keeping the helper. I think we've done this in some places. It has the "helper" but in the end it can just take in a string.

Reply:
Probably we could make it a string. But then we need to properly detect the actual type that could be anything, and define the base64 part, eg `data:image/png;base64`. It's easier if we have only a few predefined types.